cli/dcoscli/test/package.py

```diff
@@ -48,30 +52,25 @@ def package(package_name, deploy=False, args=[]):
 
 def setup_universe_server():
     # add both Unvierse servers with static packages
-    add_app('tests/data/universe-v3-stub.json', True)
-    add_app('tests/data/helloworld-v3-stub.json', True)
+    add_app('tests/data/universe-v3-stub.json', False)
+    add_app('tests/data/helloworld-v3-stub.json', False)
 
     assert_command(
         ['dcos', 'package', 'repo', 'remove', 'Universe'])
 
+    watch_all_deployments()
+
     # Add the two test repos to Cosmos
     for name, url in UNIVERSE_TEST_REPOS.items():
-        assert_command(['dcos', 'package', 'repo', 'add', name, url])
+        # wait for DNS records for the universe app to be propagated
```

Comment:
This looks like a completely different strategy from before for waiting for the universe to come online. What's better about this way (or different at least)?

Reply:
`pacakge repo add` now fails if the universe service is unreachable, see see https://mesosphere.slack.com/archives/C02H2798B/p1512426126000076